packages/backup-restore-operator/charts/values.yaml

```diff
@@ -0,0 +1,61 @@
+image:
+  repository: rancher/backup-restore-operator
+  tag: v0.0.1-rc9
+
+## If set, the rancher-resource-set CR will be created in this namespace. Otherwise it will be created in the release namespace
+## Any Backups created using this resourceSet must be created in the same namespace.
+rancherResourceSetNamespace: ""
+
+## Default s3 bucket for storing all backup files created by the backup-restore-operator
+s3:
+  enabled: false
+  ## credentialSecretName if set, should be the name of the Secret containing AWS credentials. It should be created in the Chart's namespace and can be created after the chart is installed. 
+  ## To use IAM Role, don't set this field
+  credentialSecretName: creds 
+  region: us-west-2
+  bucketName: rancherbackups
+  folder: base folder
+  endpoint: s3.us-west-2.amazonaws.com
+  endpointCA: base64 encoded CA cert
+  # insecureTLSSkipVerify: optional
+
+## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+## If persistence is enabled, operator will create a PVC with mountPath /var/lib/backups
+persistence: 
+  enabled: false
+
+  pvc:
+    enabled: false
+    
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack). 
+    ## Refer https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1
+    ##
+    storageClass: "-"
+
+    ## Only certain StorageClasses allow resizing PVs; Refer https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/
+    size: 2Gi
+
+  nfs:
+    enabled: false
+   
+    ## Path that is exported by the NFS server. More info: https://kubernetes.io/docs/concepts/storage/volumes#nfs
+    path: 
+    
+    ## Server is the hostname or IP address of the NFS server. More info: https://kubernetes.io/docs/concepts/storage/volumes#nfs
+    server: 
```

Comment:
nit:
```suggestion
    ## Path that is exported by the NFS server. More info: https://kubernetes.io/docs/concepts/storage/volumes#nfs
    path: ""
    
    ## Server is the hostname or IP address of the NFS server. More info: https://kubernetes.io/docs/concepts/storage/volumes#nfs
    server: ""
```

Reply:
not applicable anymore, after some discussion we decided to remove nfs part